src/interface/src/app/features/feature.guard.ts

```diff
@@ -1,7 +1,7 @@
 import { inject } from '@angular/core';
 import { CanActivateFn, CanMatchFn, Router } from '@angular/router';
 
-import { FeatureService } from './feature.service';
+import { FeatureService} from './feature.service';
```

Comment:
nit: spacing (to make consistent with other imports { FeatureService } vs { FeatureService})